ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -2150,7 +2150,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
this is unwanted, and here just for kick off the jobs. Once we know there is no regression we'll remove that change here.